### PR TITLE
add isin to jax.numpy logic functions

### PR DESCRIPTION
**Pull request by AntoniosMalak**

### CODE REVIEW

File: ivy/functional/frontends/jax/numpy/logic.py [hunk 1]
```diff
@@ -153,6 +153,11 @@ def isfinite(x, /):
     return ivy.isfinite(x)
 
 
+@to_ivy_arrays_and_back
+def isin(element, test_elements, assume_unique=False, invert=False):
+    return ivy.isin(element, test_elements, assume_unique=assume_unique, invert=invert)
```
Comment (jkeane508) [inline ivy/functional/frontends/jax/numpy/logic.py:158]: Hello, so this Ivy function is currently in the experimental API  which means that it can't be called until it's been revised and moved into the functional API.

In order for this to be able to merge, you will need to implement ivy.isin() here with a composition of current functional API functions.

Hint: This function has already been implemented as a composition of Ivy functions 

Thanks,
James

**Review by jkeane508 — CHANGES_REQUESTED** [review]

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
